spec/definitions/prepair_host.sh

```diff
@@ -0,0 +1,11 @@
+#update git
+cd ~/machinery/ && git pull
+
+#build package
+cd ~/machinery/ && rake rpm:build[https://api.suse.de,Devel:Alfred,SLE_12]
+
+#install package
+package=$(find ~/machinery/package/ \
+  -name "machinery-[0-9]*.ppc64le.rpm" -or \
+  -name "machinery-[0-9]*.s390x.rpm")
```

Comment:
Why do we need to list the architectures here? Can't we do that with a wildcard?